core/src/chiplets/mod.rs

```diff
@@ -24,3 +24,28 @@ pub const HASHER_TRACE_OFFSET: usize = CHIPLETS_OFFSET + NUM_HASHER_SELECTORS;
 pub const BITWISE_TRACE_OFFSET: usize = CHIPLETS_OFFSET + NUM_BITWISE_SELECTORS;
 /// The first column of the memory chiplet.
 pub const MEMORY_TRACE_OFFSET: usize = CHIPLETS_OFFSET + NUM_MEMORY_SELECTORS;
+
+// --- GLOBALLY-INDEXED CHIPLET COLUMN ACCESSORS --------------------------------------------------
+
```

Comment:
Not for this PR, but I wonder if we should add diagrams of how the trace looks like here (similar to how we do in other places).

Reply:
Yeah, I think that's a good idea